lookml/orders.view.lkml

```diff
@@ -70,51 +95,254 @@ view: orders {
+    type: number
+    sql: ${TABLE}.customer_first_90_day_completed_orders ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: customer_first_90_day_revenue {
+    description: "This is a user's sum of total amount in the first 90 days after their first order"
+    type: number
+    sql: ${TABLE}.customer_first_90_day_revenue ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: customer_type {
+    description: "This classifies a customer based on their purchasing ('single_purchaser', 'repeat_purchaser', 'non_purchaser')"
+    type: string
+    sql: ${TABLE}.customer_type ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: days_from_first_completed_order {
+    description: "This returns the number of days between the order placed and a customer's first completed order date"
+    type: number
+    sql: ${TABLE}.days_from_first_completed_order ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: days_since_previous_completed_order {
+    description: "This returns the number of days between the order placed and a customer's last completed order"
+    type: number
+    sql: ${TABLE}.days_since_previous_completed_order ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: weeks_from_first_completed_order {
+    description: "This is a count of weeks since a customer's first completed order"
+    type: number
+    sql: ${TABLE}.weeks_from_first_completed_order ;;
+  }
+
+  dimension: lifetime_completed_orders {
+    description: "This is a user's count of completed orders all time"
+    type: number
+    sql: ${TABLE}.lifetime_completed_orders ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: lifetime_completed_revenue {
+    description: "This is a user's total sum of completed revenue all time"
+    type: number
+    sql: ${TABLE}.lifetime_completed_revenue ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: lifetime_placed_orders {
+    description: "This is a user's count of orders all time"
+    type: number
+    sql: ${TABLE}.lifetime_placed_orders ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: lifetime_revenue {
+    description: "This is a user's total sum of revenue all time"
+    type: number
+    sql: ${TABLE}.lifetime_revenue ;;
+    group_label: "Customer Attributes"
+  }
+
+  dimension: months_from_first_completed_order {
+    description: "This is a count of months since a customer's first completed order"
+    type: number
+    sql: ${TABLE}.months_from_first_completed_order ;;
+    group_label: "Customer Attributes"
   }
 
   dimension: new_vs_repeat {
+    description: "This returns 'new' for the customer's first placed order and 'repeat' for subsequent orders"
     type: string
-    sql: ${TABLE}.new_or_repeat ;;
+    sql: ${TABLE}.new_vs_repeat ;;
+    group_label: "Customer Attributes"
   }
 
-  dimension: months_from_start {
+#------------------------------------------ DIMENSIONS
+
+  dimension: billing_address_id {
```

Comment:
I think you should remove billing shipping and location id - these are shopify specific